orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/job/JobRunner.java

```diff
@@ -0,0 +1,37 @@
+package com.netflix.spinnaker.orca.clouddriver.tasks.job;
```

Comment:
converted to java to get a `default` interface implementation for `getJobTimeoutMs`